### DISCUSSION TITLE
How to compile mxnet 1.8 with HDFS supported?

### DISCUSSION THREAD

**Discussion by Xiaoxiong-Liu** [apache/incubator-mxnet#20154] [Q&A] [answered]
There is no option for hdfs support in [cmake file](https://github.com/apache/incubator-mxnet/blob/v1.x/config/linux_gpu.cmake)?
So how can I turn ON hdfs when compiling?
thank you .

**Comment by leezu** [comment, marked as answer]
It's not an option of mxnet, but HDFS support is provided by the dmlc-core library. You can just set `-DUSE_HDFS=1` when compiling MXNet and the option will be correctly set at https://github.com/dmlc/dmlc-core/blob/4cabfeafb4f8f8e574de89cea0780c993c20c2d8/CMakeLists.txt#L16 as dmlc-core is compiled during the mxnet compilation.
